### PR TITLE
feat(checkout): Add support for Google Pay passes

### PR DESCRIPTION
**Pull request by christopherdro**
Screen Shot 2022-03-25 at 5 00 17 PM

### CODE REVIEW

File: packages/app/pages/api/ethpass.ts [hunk 1]
```diff
@@ -28,10 +28,18 @@ export default function handler(req: NextApiRequest, res: NextApiResponse) {
...
+            platform,
+            templateId: process.env.ETHPASS_TEMPLATE_ID,
+            barcode: { message: "Thanks for participating at Cabin’s first inaugural GUILD GAMES!"},
+            pass: {   
               description,
+              headerFields: [{
+                key: "header1",
+                value: "DEMO PASS",
+                textAlignment: "PKTextAlignmentNatural"
+              }],
```
Comment (dvargasfuertes) [inline packages/app/pages/api/ethpass.ts:40]: what does this do?

**Review by dvargasfuertes — APPROVED** [review]
Hell yea, thanks for this!

File: packages/app/pages/api/ethpass.ts (hunk 1 above)
Comment (christopherdro) [reply packages/app/pages/api/ethpass.ts:40]: IMG_163F5536D22E-1

Comment (christopherdro) [reply packages/app/pages/api/ethpass.ts:40]: It's an empty canvas right now so we can decide what other info we want to put here. I was thinking we can pull the name and symbol from the contract as a start.

I didn't get around to customizing the Google pass quite yet so it's gonna look like the one from our site. I'll work on it starting next week and ping all of you for some feedback.

**Pull request merged**